datafeeder/frontend-config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "encodings": [
+    {
+      "label": "UTF-8",
+      "value": "UTF-8"
+    },
+    {
+      "label": "ISO-8859-1",
+      "value": "ISO-8859-1"
+    }
+  ],
+  "projections": [
+    {
+      "label": "WGS84",
+      "value": "EPSG:4326"
+    },
+    {
+      "label": "Lambert 93",
+      "value": "EPSG:2154"
+    },
+    {
+      "label": "Web Mercator",
+      "value": "EPSG:3857"
+    }
+  ],
+  "thesaurusUrl": "https://georchestra.mydomain.org/geonetwork/srv/api/registries/vocabularies/search?type=CONTAINS&thesaurus=external.theme.httpinspireeceuropaeutheme-theme&rows=200&q=${q}&uri=**&lang=fre"
```

Comment:
After having played the ansible playbook, a INSPIRE theme thesaurus is already available, but under "external.theme.inspire-theme".